src/CalcManager/CalculatorManager.cpp

```diff
@@ -117,6 +117,15 @@ namespace CalculationManager
         m_displayCallback->SetParenDisplayText(parenthesisCount);
     }
 
+    /// <summary>
+    /// Callback from the engine
+    /// Used to set the narrator text when no parenthesis can be added
+    /// </summary>
+    void CalculatorManager::OnNoParenAdded()
```

Comment:
Should this be `OnNoParenAdded` or `OnNoRightParenAdded`?  Are there cases where we don't add a left paren?